benchbuild/engine.py

```diff
@@ -0,0 +1,47 @@
+"""
+Orchestrate experiment execution.
+"""
+import typing as tp
+
+import attr
```

Comment:
Codacy found an issue: [Unable to import 'attr'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)